src/vs/workbench/contrib/notebook/browser/contrib/editorHint/emptyCellEditorHint.ts

```diff
@@ -84,6 +79,11 @@ export class EmptyCellEditorHintContribution extends EmptyTextEditorHintContribu
 			return false;
 		}
 
+		const shouldRenderHint = super._shouldRenderHint();
```

Comment:
@rebornix This was a simpler fix, as you can see in the previous line we're already trying to find the active notebook editor.